lib/more_core_extensions/core_ext/string/decimal_suffix.rb

```diff
@@ -0,0 +1,19 @@
+module MoreCoreExtensions
+  module DECIMAL_SI
+    # made to handle https://github.com/kubernetes/kubernetes/blob/4def5add114b651c26fe576b7315f8029bfce46a/vendor/github.com/appc/spec/schema/types/resource/quantity.go#L49
```

Comment:
I don't think we need a kubernetes reference comment for something that is supposed to be generic.